### ISSUE TITLE
`gutterIconPath` is not shown in High Contrast theme

### ISSUE THREAD

**Issue by alefragnani** [Microsoft/vscode#3642]
I'm creating a `TextEditorDecorationType`, defining `gutterIconPath` with a `.png` file. It works fine for all stock themes, but it is not shown when using the _High Contrast_ theme.

Is there any _format/setting_ for the image to work with _High Contrast_ theme?

The image is [this one](https://raw.githubusercontent.com/alefragnani/vscode-bookmarks/master/images/bookmark.png)

Thanks in advance.

**Comment by bgashler1** [comment]
@alefragnani: this is probably due to it being displayed as a background image.  We use a lot of `::after`/`::before` pseudo elements to force icons to display in high-contrast mode on Windows, but we're going o get away from that soon and use a better method.

Could you web inspect using **Developer Tools** in the help menu to where it should be showing up while in the high-contrast theme.  I'm curious if there's a `background: none` that's wiping it out in high contrast theme.

**Comment by alefragnani** [comment]
Hi @bgashler1 ,

I don't see any `background: none` in there, I'm sorry. But maybe is my fault, because I'm just starting to play with CSS and the Chrome Developer Tools :smile: .

But I found something weird in _Inspect_ while switching themes, that could help. This image below shows the _normal theme_ and the next one, the _high contrast_. I noted that when I choose the _high contrast_, an element (`media: "screen"`) is removed. The _breakpoint_ icon, on the other hand, still works in _High Contrast_, and a similar element to that `media: "screen"` is not removed. I hope this could say something :)

![guttericon missing - webinspect - normal theme](https://cloud.githubusercontent.com/assets/3781424/13657629/7efba586-e64f-11e5-8e89-e7becab81cf7.png)

![guttericon missing - webinspect - highcontrast](https://cloud.githubusercontent.com/assets/3781424/13657784/b433f5ae-e650-11e5-9f6a-9ca0e924a4d7.png)

By the way, the problem is happening with my **Bookmarks** extension. Maybe installing it could also help you inspecting the CSS.

Thanks in advance

**Comment by bgashler1** [comment]
Just found a clue to what may be causing it.

**codeEditorServiceImpl.ts**
```
	private static _getSelectorPrefixOf(theme:ThemeType): string {
		if (theme === ThemeType.Light) {
			return '.monaco-editor.vs';
		}
		return '.monaco-editor.vs-dark';
	}
```

Changing the above `vs-dark` to `hc-black` (line 250) causes your icon to appear in the high-contrast theme.  This leads me to believe that we are treating hc-black theme as neither light or dark, so it's never getting your image.

I'm assigning to our development team to investigate.
